grepdemos/cal.com
# chore: Add summaries for all API v2 endpoints

## What does this PR do?

Since we are rolling out new docs, we need to have valid summaries on all endpoints.

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Run the API and go to /docs-json to ensure the summaries are generated properly
